src/http.c

```diff
@@ -597,7 +597,7 @@ int auth_log_in(struct tunnel *tunnel)
 	char reqid[32] = { '\0' };
 	char polid[32] = { '\0' };
 	char group[128] = { '\0' };
-	char data[1024], token[128], tokenresponse[256];
+	char data[1024], token[128], tokenresponse[256], tokenparams[320];
```

Comment:
I tried that; but I got a compiler warning when trying to fill `tokenresponse` on line 696:
```
snprintf(tokenparams, sizeof(tokenparams),
	         "code=%s&code2=",
	         tokenresponse);
```
If I recall correctly; the compiler emitted a warning that possibly a too-large amount of data would be put in...

But I'm not that well-versed in C++, so it might very well be possible :).

Reply:
Perhaps I can try to fix this for you - just increase the size of `tokenresponse` from 256 to 320. I shall modify your branch directly, it's usually possible in the context of a pull request. WIll you then review my changes and check they still work?

Reply:
If I understood the warning correctly; doing the following will always complain:
```
			url_encode(tokenresponse, cfg->otp);
			snprintf(tokenresponse, sizeof(tokenresponse),
			         "code=%s&code2=",
			         tokenresponse);
```

The compiler complains with:
```
src/http.c:696:13: warning: passing argument 1 to restrict-qualified parameter aliases with argument 4 [-Wrestrict]
  696 |    snprintf(tokenresponse, sizeof(tokenresponse),
      |             ^~~~~~~~~~~~~
  697 |             "code=%s&code2=",
  698 |             tokenresponse);
      |             ~~~~~~~~~~~~~
src/http.c:697:19: warning: ‘%s’ directive output may be truncated writing up to 255 bytes into a region of size 251 [-Wformat-truncation=]
  697 |             "code=%s&code2=",
      |                   ^~
  698 |             tokenresponse);
      |             ~~~~~~~~~~~~~
```

This is why I added the third field, as I had no idea how to otherwise solve this :(. Increasing the size of `tokenresponse` will only change the truncation bytes.

Reply:
Ah right, it looks like we do need two variables anyway.

Then there's just the documentation issue only, new options need to be documented in `src/main.c` and `doc/openfortivpn.1.in`.